src/zac/demo/mor/tests/tests.py

```diff
@@ -0,0 +1,105 @@
+import collections
+import json
+import requests
+from django_webtest import WebTest
+from time import sleep
+
+from django.conf import settings
+from django.urls import reverse
+from django.utils import timezone
+from django.utils.translation import gettext
+
+from ...models import SiteConfiguration
+
+from .factories import *
+
+
+api_scope = 'api/v1/'
+
+
+def get_user_json(user):
+
+    return json.dumps({
+        'username': user.username,
+        'password': user.password
+    })
+
+
+class TestViews(WebTest):
+
+    json_header = {
+        'Content-Type': 'application/json'
+    }
+
+    def get_test_zaak(self):
+        return json.dumps({
+            "bronorganisatie": "509381406",
+            "registratiedatum": "2018-12-06",
+            "startdatum": "2018-12-06",
+            "toelichting": "Hier ben ik",
+            "verantwoordelijkeOrganisatie": "245122461",
+            "zaakgeometrie": {"coordinates": [4.891362190246582, 52.3731887966551],
+                              "type": "Point"},
+            "zaaktype": "{{zaaktype}}"
+        })
+
+    def setUp(self):
+        self.test_url = settings.TEST_PLATFORM_URL
+        self.config = SiteConfigurationFactory()
```

Comment:
will break if there's already a `SiteConfig` object. Use `SiteConfig.get_solo()` which will `get_or_create` it.

You also need to add a `self.addCleanup(cache.clear)` to the setup, because django-solo caches the config.